### ISSUE TITLE
Cannot comminicate with MATLAB when using public IP in Jupyter

### ISSUE THREAD

**Issue by flototh** [mathworks/jupyter-matlab-proxy#92]
When using an externally accessible IP when starting Jupyter, e.g. `jupyter lab --ip xx.xx.xx.xx`, I get the error
```
ERROR:MATLABKernel.fd50cfab-30d9-4841-b2d2-6e95b3e0a10c:Found a startup error: 
                Error: MATLAB Kernel could not communicate with MATLAB.
                Reason: Possibly due to invalid jupyter security tokens.
```

upon executing any Matlab command in a Matlab notebook.

When using the default `localhost` everything works fine.

The use case is to access Matlab on a remote server via the web browser. While this works fine with `localhost` and suitable ssh tunnels, using a "outwards" facing IP for the jupyter process would make things easier. Is this behavior supported?

**Comment by rashedmyt** [comment]
Hi @flototh 

Thank you for using MATLAB Integration for Jupyter.

I was able to reproduce the issue and we are working on fixing this bug in the upcoming releases of `jupyter-matlab-proxy`

**Comment by flototh** [comment]
Great to hear & looking forward to the fix. Thanks for sharing this awesome software!

**Comment by prabhakk-mw** [comment]
fixed by https://github.com/mathworks/jupyter-matlab-proxy/commit/1648aafdb66a5879b1461f37addbe30b6fbb9281 and is available as a part of [v0.12.3](https://pypi.org/project/jupyter-matlab-proxy/0.12.3/)

Kindly let us know if this issue persists after upgrading your package to v0.12.3 using the following command in your python environment:
```bash
python -m pip install -U jupyter-matlab-proxy
```
